src/libs/actions/Task.ts

```diff
@@ -833,42 +867,31 @@ function dismissModalAndClearOutTaskInfo() {
 
 /**
  * Returns Task assignee accountID
- *
- * @param {Object} taskReport
- * @returns {Number|null}
  */
-function getTaskAssigneeAccountID(taskReport) {
+function getTaskAssigneeAccountID(taskReport: OnyxTypes.Report): number | undefined {
     if (!taskReport) {
-        return null;
+        return;
     }
 
     if (taskReport.managerID) {
         return taskReport.managerID;
     }
 
     const reportAction = ReportActionsUtils.getParentReportAction(taskReport);
-    return lodashGet(reportAction, 'childManagerAccountID');
+    return reportAction.childManagerAccountID;
 }
 
 /**
  * Returns Task owner accountID
- *
- * @param {Object} taskReport
- * @returns {Number|null}
  */
-function getTaskOwnerAccountID(taskReport) {
-    return lodashGet(taskReport, 'ownerAccountID', null);
+function getTaskOwnerAccountID(taskReport: OnyxTypes.Report): number | null {
```

Comment:
In the function above you changed to return to `undefined`. Let's standardise to either `null` or `undefined`.